en/theme/material/assets/css/redoc.css

```
@@ -0,0 +1,198 @@
+.api-content > div > div > div:first-child  {
+    background: var(--md-default-bg-color) !important;
+    padding: 0px 25px;
+}
+
+.api-content > div > div > div:last-child h3 {
+    color: #fff !important;
+}
+
+.api-content td div[class="sc-ikkxIA daqcVd"] {
+    background-color: var(--md-default-bg-color) !important;
+    color: var(--md-default-fg-color--light) !important;
```

Stylelint (17.2.0)

[error] Expected custom property name "--md-default-fg-color--light" to be kebab-case (custom-property-pattern)

(custom-property-pattern)